src/components/ChatEntry.js

```diff
@@ -1,22 +1,69 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+
 
 const ChatEntry = (props) => {
+  const [likeButton, setLikeButton] = useState('🤍');
+
+  const onLikedButtonClick = () => {
+    const updatedMessage = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked,
+  
+  // let senderClass = 'chat-entry local'
+  // if (sender === 'Estragon') {
+  //   senderClass = 'chat-entry remote'
+}
+      props.updateMessageData(updatedMessage);
+      setLikeButton(updatedMessage.liked ? '❤️' : '🤍');
+      const bool = likeButton === '🤍';
+      props.updateTotalLikes(bool);
+  };
+
+  const localRemote =
+    props.sender === 'Vladimir' ? 'chat-entry local' : 'chat-entry remote';
```

Comment:
NIce way to figure out whether to apply the local or remote css class. But what if the chat participants were other senders? Could we determine the "local sender" higher up and then pass that information down so that we can avoid hard-coding a name?